src/features/movies/MovieDetails/index.js

```diff
@@ -0,0 +1,89 @@
+import MainWrapper from "../../../common/MainWrapper";
+import {
+  DetailsImage,
+  DetailsText,
+  DetailsTitle,
+  DetailsWrapper,
+  TextWrapper,
+  Year,
+  Info,
+  RatingWrapper,
+  StarIcon,
+  Rate,
+  Votes,
+  Tag,
+  TagsWrapper,
+  AdditionalInfoWrapper,
+} from "../../../common/Details";
+import { PosterWrapper, PosterDisplay, MainInfoWrapper, MainInfoTitle } from "../../movies/MovieDetails/styled";
+import PeopleList from "../../people/PeopleList";
+import image from "../../../common/images/poster.png";
+
+const MovieDetails = () => {
+  return (
+    <>
+      <PosterDisplay>
+        <PosterWrapper>
+          <MainInfoWrapper>
+            <MainInfoTitle>Mulan long title</MainInfoTitle>
+            <RatingWrapper>
+              <StarIcon infoMain />
+              <Rate infoMain >7,8</Rate>
+              <Votes infoMain >/ 10</Votes>
+              <Votes infoMain normalHidden>335 votes</Votes>
+            </RatingWrapper>            
+            <RatingWrapper mobileHidden>
```

Comment:
I have error in the consol with `infomain`.

Reply:
For mobile 320px is difference versus Figma (In bigposter, In cast and crew - number of tiles we need 2 columns but now we have one)

Reply:
For desktop I can see 5 columns with tiles but in Figma we have 6 columns (cast and crew).

Reply:
All errrors corrected.

Reply:
For 320px is still one column cast people in movie details instead of two.

Reply:
Fixed